types.d.ts

```diff
@@ -84,6 +84,7 @@ import {
 	Extend,
 	ValidationErrorConfiguration
 } from "schema-utils/declarations/validate";
+import { RawSourceMap } from "source-map/source-map";
```

Comment:
This might be a problem, since we do not depend on source-map (and don't want to)...